app/assets/scripts/components/Project.js

```diff
@@ -65,6 +65,9 @@ class Project extends React.Component {
         return key !== 'name';
       }).map(function (key) {
         if (!keys[key]) { return <div></div>; }
+        if (keys[key].type === 'object' && 'ar' in keys[key].properties) {
```

Comment:
keys[key] refers to an entry in the schema properties [here](https://github.com/map-egypt/admin/blob/master/app/assets/scripts/components/ProjectForm.js#L8)